### PR TITLE
New case event fields

### PR DESCRIPTION
**Pull request by AdeHarrison**
# How to test?
Check out and build a local docker image from this branch
[Test as part of](https://github.com/ONSdigital/census-rm-case-processor/pull/26)

# Links
[Trello](https://trello.com/c/VhVMCJ2M/813-additional-columns-for-case-events-table-8)

# Screenshots (if appropriate):

### CODE REVIEW

**Comment by AdeHarrison** [comment]
[Additional columns for Case events table ** (8)](https://trello.com/c/VhVMCJ2M/813-additional-columns-for-case-events-table-8)

File: app/rabbit_helper.py [hunk 1]
```diff
@@ -50,10 +50,11 @@ def send_message_to_rabbitmq(message,
     """
     rabbitmq_connection = _create_connection()
     rabbitmq_channel = rabbitmq_connection.channel()
+    hdrs = {'source': 'RECEIPTING', 'channel': 'EQ'}
```
Comment (NickGrantX) [inline app/rabbit_helper.py]: maybe call it headers we're not short of characters

**Review by NickGrantX — APPROVED** [review]
Minor comment but looks good 👍

File: app/rabbit_helper.py (hunk 1 above)
Comment (AdeHarrison) [reply app/rabbit_helper.py]: changed

**Review by LukeLoze1 — APPROVED** [review]

**Pull request merged**